### DISCUSSION TITLE
关于DownKyi.exe

### DISCUSSION THREAD

**Discussion by zbn2022** [leiurayer/downkyi#532] [Q&A]
完全找不到这个文件在哪里打开，有高人能指点一下吗？

**Comment by cocos12345** [comment]
您好，在code页面直接download zip会下载项目的源代码，如果您仅仅想安装项目，请在code界面下拉，在下载一栏中点击下载release版本，在release版本文件夹中点击DownKyi.exe即可运行本项目
